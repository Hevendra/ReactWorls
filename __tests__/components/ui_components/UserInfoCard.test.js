import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import renderer from 'react-test-renderer'
import UserInfoCard  from '../../../src/components/ui_components/UserInfoCard'

describe('UserInfoCard Component',()=>{
    it('matches the snapshot',()=>{
        const tree=renderer.create(<UserInfoCard/>).toJSON
        expect(tree).toMatchSnapshot()
    })
})

