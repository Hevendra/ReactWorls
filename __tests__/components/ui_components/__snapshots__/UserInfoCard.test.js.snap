// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserInfoCard Component matches the snapshot 1`] = `[Function]`;
